### PR TITLE
etcdserver:set FDUsage ticker to 10 minute from 5 seconds.

### LINKED ISSUE #11969: etcdserver: runtime.FDUsage() function  high CPU usage, create large numbers of objects
# version
etcd Version: 3.3.17
Git SHA: 6d80523
Go Version: go1.12.9
Go OS/Arch: linux/amd64

# issue

* When the number of etcdserver clients reaches 100K, there will be more than 100K fds under `/proc/self/fd` in the etcd process.
* The `monitorFileDescriptor` function will execute the `runtime.FDUsage()` function per 5 seconds.
* The `runtime.FDUsage` function scans all directories and files under `/proc/self/fd` and returns `[]os.FileInfo`.

# result:
* A lot of fd scans will bring high CPU usage.
* A lot of of fd reads will bring a lot of application and allocation of memory objects.

cpu_fdusage
fdusage_all_objects

# solution:
* Delete the acquisition of `FDUsage` and cancel the comparison with the `syscall.Rlimit`. Error message that depends on fd operation failure.
* Extend the execution time of the monitorFileDescriptor function from every 5 seconds to 24hours.

@gyuho @xiang90 @jpbetz @jingyih @spzala

### PR DESCRIPTION
**Pull request by cfc4n**
Fixes #11969

### CODE REVIEW

File: etcdserver/metrics.go [hunk 1]
```diff
@@ -184,7 +184,7 @@ func init() {
 }
 
 func monitorFileDescriptor(lg *zap.Logger, done <-chan struct{}) {
-	ticker := time.NewTicker(5 * time.Second)
+	ticker := time.NewTicker(1 * time.Hour)
```
Comment (xiang90) [inline etcdserver/metrics.go:187]: One hour seems to be too long.

Comment (cfc4n) [reply etcdserver/metrics.go:187]: > One hour seems to be too long.
What about 10 minute?

**Comment by cfc4n** [comment]
PTAL ,thanks @spzala

**Comment by gyuho** [comment]
@cfc4n Can you inline comment this change in addition to linking the github issue? otherwise, lgtm :)

**Comment by cfc4n** [comment]
> @cfc4n Can you inline comment this change in addition to linking the github issue? otherwise, lgtm :)

Done, Thanks for review.

**Comment by gyuho** [comment]
lgtm thx

**Comment by dims** [comment]
@cfc4n @xiang90 @gyuho please see the change made in tailscale for the same problem - https://github.com/tailscale/tailscale/pull/2785

**Pull request merged**